src/Skynet.Server/Services/PacketService.cs

```diff
@@ -98,7 +99,7 @@ private static Type[] MapHandlers(Packet[] packets)
             for (int i = 0; i < packets.Length; i++)
             {
                 Packet packet = packets[i];
-                if (packet == null || !packet.Policies.HasFlag(PacketPolicies.Receive))
+                if (packet == null || !packet.Policies.HasFlag(PacketPolicies.ClientToServer))
```

Comment:
Just a suggestion to make the enum names shorter: Maybe Uplink (C->S) and Downlink (S->C) instead of ClientToServer and ServerToClient?

Reply:
This enum is implemented in [`Skynet.Protocol`](https://github.com/skynet-im/skynet-dotnet/blob/824875a3592a32fd6f260acb5bc3a583f621634c/src/Skynet.Protocol/PacketPolicies.cs). Earlier we had the names `Send` and `Receive` which was fine for server-sided development. Now, the same enum has to be used for server and client and I wanted to have more descriptive names even if they are longer.